firebase-auth/src/androidMain/kotlin/dev/gitlive/firebase/auth/credentials.kt

```diff
@@ -104,6 +101,9 @@ actual class PhoneAuthProvider(val android: com.google.firebase.auth.PhoneAuthPr
                 }
             }
 
+            override fun onCodeAutoRetrievalTimeOut(verificationId: String) {
+            }
+
             override fun onVerificationCompleted(credential: com.google.firebase.auth.PhoneAuthCredential) {
                 response.complete(Result.success(AuthCredential(credential)))
```

Comment:
i know this isnt your code but we shoudlnt be wrapping the result in a Result - can just fail the CompletableDeferred with [completeExceptionally](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-completable-deferred/complete-exceptionally.html)

Reply:
So instead of response.complete(Result.failure(exception)) we should do response.completeExceptionally(exception) ?